src/main/java/app/linguistai/bmvp/service/gamification/UserStreakService.java

```diff
@@ -129,7 +146,7 @@ public RUserStreak getUserStreakByToken(String email) throws Exception {
             throw e1;
         }
         catch (Exception e2) {
-            System.out.println("ERROR: Could not fetch UserStreak.");
+            System.out.println("ERROR: Could not fetch UserStreak."); // TODO what is this
```

Comment:
please remove todo (discussed already, will be addressed with [LINGUIST-350](https://linguistai.atlassian.net/browse/LINGUIST-350))